bin/check-http.rb

```diff
@@ -205,8 +205,8 @@ class CheckHttp < Sensu::Plugin::Check::CLI
          proc: proc(&:to_i)
 
   option :response_code,
-         long: '--response-code CODE',
-         description: 'Check for a specific response code'
+         long: '--response-code REGEX',
+         description: 'Check for a specific response code REGEX'
```

Comment:
This does not convey if the matching or unmatched HTTP status codes will have an OK status. Can we add something to clarify?